cypress/e2e/authorization.cy.ts

```diff
@@ -8,9 +8,9 @@ describe('Authorization', () => {
     cy.location('pathname').should('eq', '/login');
   });
 
-  // NOTE this is in scope of FSK-97
-  it.skip('should be redirected to the families page after successful login', () => {
+  it('should be redirected to the families page after successful login', () => {
     cy.login();
+    cy.visit('/families');
```

Comment:
You should not visit the families page, login should redirect you to the families page.